packages/types/test/unit/ssz.test.ts

```diff
@@ -0,0 +1,54 @@
+import {expect} from "chai";
+import {ssz} from "../../src";
+
+describe("size", function () {
+  it("should calculate correct minSize and maxSize", () => {
+    const minSize = ssz.phase0.BeaconState.minSize();
+    const maxSize = ssz.phase0.BeaconState.maxSize();
+    // https://gist.github.com/protolambda/db75c7faa1e94f2464787a480e5d613e
+    expect(minSize).to.be.equal(2687377);
+    expect(maxSize).to.be.equal(141837543039377);
+  });
```

Comment:
This test is not necessary here

Reply:
that was an old test (renamed size.test.ts to this file)

Reply:
Yes please remove, this test may go stale